core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -176,11 +186,17 @@ oppia.controller('FeedbackTab', [
             var UNSAVED_CHANGES_MSG = 'You have unsaved changes to ' +
               'this exploration. Please save/discard your unsaved changes if ' +
               'you wish to accept.';
+            var ACTION_EDIT_SUGGESTION = 'edit';
+            $scope.stateName = stateName;
+            $scope.suggestion_id = suggestionId;
+            $scope.target_id = targetId;
             $scope.isNotHandled = !suggestionIsHandled;
             $scope.canEdit = EditabilityService.isEditable();
             $scope.canReject = $scope.canEdit && $scope.isNotHandled;
             $scope.canAccept = $scope.canEdit && $scope.isNotHandled &&
               suggestionIsValid && !unsavedChangesExist;
+            $scope.suggestionEditorIsShown = false;
+            $scope.saveButton = false;
```

Comment:
I did not understand the use of this variable through its name.

Reply:
```suggestionEditorIsShown``` is used to check whether we have to show suggestion editor or not.
Or in simple words, as we have to edit suggestions for submit suggestion there is a suggestion editor which allows to edit suggestion. And suggestion editor is shown only when user clicks on ``` edit ``` buton.
Thanks!